### ISSUE TITLE
Subroutine JSON::PP::Boolean::(-- redefined with newer perls

### ISSUE THREAD

**Issue by ntyni** [makamaka/JSON-PP#76]
As discussed in https://github.com/rurban/Cpanel-JSON-XS/issues/201 this has started issuing warnings with Perl >= 5.36:

```
$ /opt/perl/bin/perl5.37.4 -we 'use Cpanel::JSON::XS (); use JSON::PP'
Subroutine JSON::PP::Boolean::(-- redefined at /opt/perl/lib/5.37.4/overload.pm line 52.
Subroutine JSON::PP::Boolean::(++ redefined at /opt/perl/lib/5.37.4/overload.pm line 52.
Subroutine JSON::PP::Boolean::(0+ redefined at /opt/perl/lib/5.37.4/overload.pm line 52.
```

I understand this should be fixed in JSON::PP.

The other direction (using JSON::PP first, then Cpanel::JSON::XS) was already fixed in Cpanel::JSON::XS 4.32.

**Comment by ntyni** [comment]
Hi, thanks for bringing this up on the Perl side. Just in case you missed it: @haarg analyzed the problem in https://github.com/Perl/perl5/issues/20246 , ending with:

> As for Cpanel::JSON::XS and JSON::PP, they should probably take advantage of overload's unimport method to remove the methods they intend to overwrite.

**Comment by charsbar** [comment]
Thanks for the notice. Shipped 4.12 with a fix suggested by haarg.
